internal/controller/falcon_node/falconnodesensor_controller.go

```diff
@@ -258,10 +260,14 @@ func (r *FalconNodeSensorReconciler) Reconcile(ctx context.Context, req ctrl.Req
 		pc := updateDaemonSetPriorityClass(dsUpdate, dsTarget, logger)
 		capabilities := updateDaemonSetCapabilities(dsUpdate, dsTarget, logger)
 		initArgs := updateDaemonSetInitArgs(dsUpdate, dsTarget, logger)
-		updated = updateDaemonSetContainerProxy(dsUpdate, nodesensor, logger)
+		proxyUpdates := updateDaemonSetContainerProxy(dsUpdate, nodesensor, logger)
+		tolsUpdate, err := r.updateDaemonSetTolerations(ctx, dsUpdate, nodesensor, logger)
+		if err != nil {
+			return ctrl.Result{}, err
+		}
 
 		// Update the daemonset and re-spin pods with changes
-		if imgUpdate || tolsUpdate || affUpdate || containerVolUpdate || volumeUpdates || resources || pc || capabilities || initArgs || initResources || updated {
+		if imgUpdate || tolsUpdate || affUpdate || containerVolUpdate || volumeUpdates || resources || pc || capabilities || initArgs || initResources || proxyUpdates || updated {
```

Comment:
`updated` should be removed, since it has been renamed as `proxyUpdates` now?

Reply:
`updated` is still used for when the configmap is updated, so it needs to stay.